src/EnergyPlus/SolarShading.cc

```diff
@@ -3947,11 +3947,11 @@ void CLIPLINE(Real64 &x1, Real64 &x2, Real64 &y1, Real64 &y2, Real64 maxX, Real6
                 else if (c2 == 3) {
                     visible = true;
                     x1 = minX;
-                    y1 = maxY + e / dx;
+                    y1 = maxY + General::SafeDivide(e, dx);
```

Comment:
There may be a better way to correct this.

Reply:
* This is fixed separately in #10657